src/components/SearchBar.js

```diff
@@ -291,17 +305,55 @@ const components = {
 };
 
 const DEFAULT_OPTIONS = [
-  { value: "nairobi", label: "Nairobi, Kenya" },
-  { value: "kisumu", label: "Kisumu, Kenya" },
-  { value: "nakuru", label: "Nakuru, Kenya" },
-  { value: "lagos", label: "Lagos, Nigeria" },
-  { value: "dar-es-salaam", label: "Dar-es-Salaam, Tanzania" },
+  { value: "kisumu", label: "Kisumu", country: "Kenya" },
+  { value: "nairobi", label: "Nairobi", country: "Kenya" },
+  { value: "nakuru", label: "Nakuru", country: "Kenya" },
+  {
+    value: "dar-es-salaam",
+    label: "Dar-es-Salaam",
+    country: "Tanzania",
+  },
+  { value: "abuja", label: "Abuja", country: "Nigeria" },
+  { value: "ilorin", label: "Ilorin", country: "Nigeria" },
+  { value: "lagos", label: "Lagos", country: "Nigeria" },
+  { value: "maiduguri", label: "Maiduguri", country: "Nigeria" },
+  {
+    value: "port-harcourt",
+    label: "Port Harcourt",
+    country: "Nigeria",
+  },
 ];
 
 function SearchBar({ handleSearch, placeholder, options, ...props }) {
   const classes = useStyles(props);
   const [single, setSingle] = useState();
 
+  // filter out unique countries
+  const countries = [...new Set(options.map((option) => option.country))];
+  const countryOptions = countries.map((country) => ({
+    value: country,
+    label: country,
+  }));
+
+  // group cities by country
+  const groupedCities = options.reduce((acc, city) => {
+    const { country } = city;
+    if (!acc[country]) {
+      acc[country] = [];
+    }
+    acc[country].push(city);
+    return acc;
+  }, {});
+
+  // create options for each country
+  const countryOptionsWithCities = countryOptions.map((option) => {
```

Comment:
```suggestion
  const countryOptionsWithCities = Object.keys(groupedCities).map((country) => {
```
I think the above will contain unique countries already without the need of `countries` or `countryOptions` above (`if (!acc[country]) {` ensures we don't duplicate `country` key).

Reply:
Awesome